### PR TITLE
Samsung internet

### PR DESCRIPTION
**Pull request by arun54321**

### CODE REVIEW

**Review by paphonb — APPROVED** [review]

**Comment by meistr91** [comment]
Can you link please samsung interneta beta too

**Comment by rodrigoswz** [comment]
> Can you link please samsung interneta beta too

Samsung Browser Beta should have its own icon since that it has a badge just like Chrome and Firefox

**Comment by meistr91** [comment]
> > Can you link please samsung interneta beta too
> 
> Samsung Browser Beta should have its own icon since that it has a badge just like Chrome and Firefox

I know it ahould have. But i think it will never happen, so im just asking to link together 😂

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -76,7 +76,8 @@
     <icon drawable="@drawable/gallery" package="com.sec.android.gallery3d" name="Gallery" />
     <icon drawable="@drawable/gallery" package="com.oneplus.gallery" name="Gallery" />
     <icon drawable="@drawable/gallery" package="com.miui.gallery" name="Gallery" />
-    <icon drawable="@drawable/browser" package="com.sec.android.app.sbrowser" name="Browser" />
+    <icon drawable="@drawable/browser" package="org.droidtr.jelly" name="Browser" />
+    <icon drawable="@drawable/sbrowser" package="com.sec.android.app.sbrowser" name="Browser" />
```
Comment (patrykmichalik) [inline app/src/main/res/xml/grayscale_icon_map.xml:80]: The app is listed on Google Play under the name Samsung Internet Browser, so I’d suggest updating the value of the `name` attribute accordingly.

```suggestion
    <icon drawable="@drawable/sbrowser" package="com.sec.android.app.sbrowser" name="Samsung Internet Browser" />
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]

**Review by patrykmichalik — APPROVED** [review]

**Comment by patrykmichalik** [comment]
Cheers, @arun54321!

**Pull request merged**